org.opent2t.sample.hub.superpopular/com.insteon.hub/js/thingTranslator.js

```diff
@@ -4,6 +4,8 @@
 "use strict";
 var request = require('request-promise');
 var OpenT2T = require('opent2t').OpenT2T;
+var OpenT2TError = require('opent2t').OpenT2TError;
+var OpenT2TConstants = require('opent2t').OpenT2TConstants;
 var sleep = require('es6-sleep').promise;
```

Comment:
Actually, the OpenT2T var shouldnt even be there as it's unneeded, and different from other translators.

---
In reply to: [103362400](https://github.com/openT2T/translators/pull/186#discussion_r103362400) [](ancestors = 103362400)